pyexasol/connection.py

```diff
@@ -729,13 +737,13 @@ def _get_login_attributes(self):
 
         return attributes
 
-    def _process_dsn(self, dsn):
+    def _process_dsn(self, dsn: str, shuffle_host_names: bool=True) -> list[ResolvedHost]:
         """
         Parse DSN, expand ranges and resolve IP addresses for all hostnames
         Return list of (hostname, ip_address, port) tuples in random order
         Randomness is required to guarantee proper distribution of workload across all nodes
         """
```

Comment:
consider updating the doc comment in regard of the optionality of the shuffling.